core-kotlin-modules/core-kotlin-7/src/test/kotlin/com/baeldung/swapfunction/classes/SwapValue.kt

```diff
@@ -0,0 +1,20 @@
+package com.baeldung.swapfunction.classes
+
+import junit.framework.TestCase.assertEquals
+
+class SwapValue {
+
+    var a = 1
+    var b = 2
+    fun swapValues(){
+        assertEquals(1, a)
+        assertEquals(2, b)
+        val temp = a
+        a = b
+        b = temp
+        assertEquals(2, a)
+        assertEquals(1, b)
+    }
+}
+
+
```

Comment:
We'll need a test import for this annotations
```suggestion
class SwapValue {

    @Test
    fun swapValues(){
        var a = 1
        var b = 2
        assertEquals(1, a)
        assertEquals(2, b)
        val temp = a
        a = b
        b = temp
        assertEquals(2, a)
        assertEquals(1, b)
    }
}

```